<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinAdminEditBI</class>
 <widget class="QMainWindow" name="WinAdminEditBI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>591</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Information actuelle</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>511</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Id Bien Immobilier : </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditIdBI">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>591</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Nouvelle information</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>511</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Adresse: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditAdresse">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Montant: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditMontant">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nombre de Pieces:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxNbrdePieces">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="text">
         <string>Type : </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="lineEditType">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Appartement</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Studio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Villa</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Details: </string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPlainTextEdit" name="plainTextEditDetails">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="text">
         <string>Statut : </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Non Occupé</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Occupé</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>590</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonModifier">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
